android/cli/commands/_build.js

```diff
@@ -2244,6 +2244,13 @@ AndroidBuilder.prototype.generateAppProject = async function generateAppProject(
 		}
 	}
 
+	const googleServicesFile = path.join(this.projectDir, 'platform', 'android', 'google-services.json');
```

Comment:
The previous for-loop copies all files in the `platform/android` directory into the `build/android/app/src/main`. So after this new code, you would have a `google-services.json` file in both `build/android/app/src/main` and `build/android/app`. Do you need the file in both places? Maybe not a bad idea to update the `afs.copyDirSyncRecursive()` call above to pass in `ignoreFiles: [ 'google-services.json' ]`.

Reply:
I talked with @jquick-axway about it and he mentioned I'd be okay to keep it for a later usage. But it's also fine to ignore it if it's really not used otherwhere.

Reply:
I'm cool either way. Thanks and hope you're doing well! Cheers!

Reply:
Cool! Same to you man, missing the unofficial architecture hangouts 😄